diff -ruwN original/debian/rules2 patched/debian/rules2
--- original/debian/rules2	2020-03-31 02:29:36.000000000 -0600
+++ patched/debian/rules2	2020-07-17 14:14:10.073981501 -0600
@@ -604,6 +604,10 @@
 endif
 
 ifneq (,$(filter $(DEB_TARGET_ARCH), amd64 i386 hurd-i386 kfreebsd-i386 kfreebsd-amd64))
+  CFLAGS += $(DEB_CFLAGS_APPEND)
+  CXXFLAGS += $(DEB_CFLAGS_APPEND)
+  CFLAGS_FOR_TARGET += $(DEB_CFLAGS_APPEND)
+  CXXFLAGS_FOR_TARGET += $(DEB_CFLAGS_APPEND)
   CONFARGS += --with-tune=generic
 endif
 
