diff -ruwN original/debian/rules2 patched/debian/rules2
--- original/debian/rules2	2020-03-31 02:29:36.000000000 -0600
+++ patched/debian/rules2	2020-07-13 11:00:17.398912600 -0600
@@ -604,8 +604,16 @@
 endif
 
 ifneq (,$(filter $(DEB_TARGET_ARCH), amd64 i386 hurd-i386 kfreebsd-i386 kfreebsd-amd64))
+ifneq (,$(POP_OPT_ARCH))
+  CFLAGS += -O3
+  CXXFLAGS += -O3
+  CFLAGS_FOR_TARGET += -O3
+  CXXFLAGS_FOR_TARGET += -O3
+  CONFARGS += --with-arch=$(POP_OPT_ARCH) --with-tune=$(POP_OPT_ARCH)
+else
   CONFARGS += --with-tune=generic
 endif
+endif
