diff -ruwN source/debian/rules patched/debian/rules
--- source/debian/rules	2020-04-14 13:26:04.000000000 -0600
+++ patched/debian/rules	2020-07-08 11:10:02.053485496 -0600
@@ -107,8 +107,8 @@
 BUILD_CC = $(DEB_BUILD_GNU_TYPE)-$(BASE_CC)
 BUILD_CXX = $(DEB_BUILD_GNU_TYPE)-$(BASE_CXX)
 
-BUILD_CFLAGS = -O2 -g
-HOST_CFLAGS = -pipe -O2 -g $(call xx,extra_cflags)
+BUILD_CFLAGS = -O2 -g $(DEB_CFLAGS_APPEND)
+HOST_CFLAGS = -pipe -O2 -g $(call xx,extra_cflags) $(DEB_CFLAGS_APPEND)
 
 # 32-bit MIPS builders have a 2GB memory space. This is not enough to
 # build test-tgmath3.o with GCC, unless tweaking the garbage collector.
diff -ruwN source/debian/testsuite-xfail-debian.mk patched/debian/testsuite-xfail-debian.mk
--- source/debian/testsuite-xfail-debian.mk	2020-04-14 13:26:04.000000000 -0600
+++ patched/debian/testsuite-xfail-debian.mk	2020-07-08 12:31:54.230197685 -0600
@@ -36,6 +36,11 @@
 test-xfail-tst-nss-files-hosts-long = yes
 test-xfail-tst-system = yes
 
+# Fails in pop-opt
+test-xfail-tst-initfinilazyfail = yes
+test-xfail-tst-latepthread = yes
+test-xfail-tst-support_descriptors = yes
+
 ######################################################################
 # alpha (including optimized flavours)
 ######################################################################
